hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ReceiverTasklet.java

```diff
@@ -205,7 +205,7 @@ int updateAndGetSendSeqLimitCompressed(long timestampNow) {
                 rwinDiff = 0;
             }
             receiveWindowCompressed += rwinDiff / 2;
-            LoggingUtil.logFinest(LOG, "receiveWindowCompressed=%d", receiveWindowCompressed);
+            LoggingUtil.logFinest(logger, "receiveWindowCompressed=%d", receiveWindowCompressed);
```

Comment:
This doesn't seem to address the concern that something should be printed only if the window size was changed